third_party/2and3/click/testing.pyi

```diff
@@ -0,0 +1,74 @@
+from typing import (IO, Any, BinaryIO, ContextManager, Dict, Iterable, List,
+                    Mapping, Optional, Tuple, Union)
+
+from .core import BaseCommand
+
+clickpkg: Any
+
+class EchoingStdin:
+    def __init__(self, input: BinaryIO, output: BinaryIO) -> None: ...
+    def __getattr__(self, x: str) -> Any: ...
+    def read(self, n: int = ...) -> bytes: ...
+    def readline(self, n: int = ...) -> bytes: ...
+    def readlines(self) -> List[bytes]: ...
+    def __iter__(self) -> Iterable[int]: ...
+
+def make_input_stream(input: IO, charset: str) -> BinaryIO: ...
+
+class Result:
+    runner: CliRunner = ...
+    stdout_bytes: bytes = ...
+    stderr_bytes: Any = ...
+    exit_code: int = ...
+    exception: Any = ...
+    exc_info: Optional[Any] = ...
+    def __init__(
+        self,
+        runner: CliRunner,
+        stdout_bytes: bytes,
+        stderr_bytes: Any,
+        exit_code: int,
+        exception: Any,
+        exc_info: Optional[Any] = ...,
+    ) -> None: ...
+    @property
+    def output(self) -> str: ...
+    @property
+    def stdout(self) -> bytes: ...
+    @property
+    def stderr(self) -> bytes: ...
+
+class CliRunner:
+    charset: Optional[str] = ...
```

Comment:
Again, no `= ...`.

Also, this one is not Optional; the constructor sets it to utf-8 if None is passed in.